utils/docker/Dockerfile.maldet.leap.15

```diff
@@ -0,0 +1,179 @@
+ARG DAOS_DEPS_BUILD=yes
+ARG DAOS_KEEP_BUILD=no
+ARG DAOS_TARGET_TYPE=release
+
+# Now do an update to ensure software is up to date for the deps build.  If the
+# src hasn't changed then this won't do anything, but if it has then we want to
+# ensure that latest dependencies are used.
+USER root:root
+RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                                                  \
+        dnf upgrade                                                                       \
+            --exclude=fuse,fuse-libs,fuse-devel,libraft0,raft-devel,mercury,mercury-devel \
+        dnf clean all;                                                                    \
+    }
+USER daos_server:daos_server
+
+ARG DEPS_JOBS=1
+
+RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                            \
+        scons --build-deps=yes --jobs $DEPS_JOBS PREFIX=/opt/daos   \
+              TARGET_TYPE=$DAOS_TARGET_TYPE --deps-only &&          \
+        ([ "$DAOS_KEEP_BUILD" != "no" ] || /bin/rm -rf build *.gz); \
+    }
+USER root:root
+
+# select compiler to use
+# Load the COMPILER arg early, and optionally install the Intel compiler
+# Do this before the CACHEBUST to make best use of cache as this is a big
+# download, and make it optional on the value of COMPILER for the same
+# reason.  Tell zypper to only consider the oneAPI repo here using the
+# discouraged --repo flag, however this prevents an update of the entire
+# upstream package list which would be immediately discarded, then reloaded
+# below in the general update section.  Only install the compilers at
+# ~900Mb rather than the entire stack at ~3Gb download.
+ARG COMPILER=gcc
+RUN if [ "$COMPILER" = "icc" ]; then                                                                     \
+        if [ -z "$REPO_FILE_URL" ] || true; then                                                         \
+            rpm --import https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB && \
+            dnf config-manager --add-repo https://yum.repos.intel.com/oneapi oneAPI;                     \
+        fi;                                                                                              \
+        dnf install intel-oneapi-compiler-dpcpp-cpp &&                                                   \
+        dnf clean all;                                                                                   \
+    fi
+
+# force an upgrade to get any newly built RPMs, but only if CACHEBUST is set.
+ARG CACHEBUST
+RUN [ -z "$CACHEBUST" ] || {                                                                         \
+        dnf upgrade --exclude=fuse,fuse-libs,fuse-devel,libraft0,raft-devel,mercury,mercury-devel && \
+        dnf clean all;                                                                               \
+    }
+
+ARG DAOS_KEEP_SRC=no
+# Remove local copy
+RUN [ "$DAOS_KEEP_SRC" != "no" ] || rm -rf /home/daos/*
+
+ARG BUILD_URL
```

Comment:
```suggestion
```

Reply:
Same as above, still need at least some of that.